src/index.html

```diff
@@ -11,7 +11,169 @@
+          <input
+            type="date"
+            name="date"
+            id="field4"
+          >
+        </div>
+        <div class="form-field">
+          <label for="field5">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="agreed"
+            id="field5"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="field6">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="field6"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="field7">Password:</label>
+          <input
+            type="password"
+            name="Password"
+            id="field7"
+            minlength="6"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+
+          <input
+            type="radio"
+            name="cats lover"
+            id="field8"
+            value="Yes"
+          >
+          <label for="field8">Yes</label>
+
+          <input
+            type="radio"
+            name="cats lover"
+            id="field9"
+            value="No"
+          >
+          <label for="field9">No</label>
+        </div>
+        <div class="form-field">
+          <label for="field10">What is your favorite color?</label>
+          <input
+            type="color"
+            name="color"
+            id="field10"
+          >
+        </div>
+        <div class="form-field">
+          <label for="field11">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="bed time"
+            id="field11"
+          >
+        </div>
+        <div class="form-field">
+          <label for="field12">What are your favorite brand of cars?</label>
+          <select
+            name="favcars"
+            id="field12"
+            multiple
+          >
+            <option value="BMW">Bmw</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+            <option value=""></option>
+          </select>
+        </div>
+        <div class="form-field">
+          <label for="field13">How do you rate our work?</label>
+          <input
+            type="range"
+            name="work rate"
+            id="field13"
+            min="0"
+            max="10"
+          >
+        </div>
+
+      </fieldset class="fieldset">
+      <fieldset>
+        <legend>Additional info:</legend>
+
+        <div class="form-field">
+          <label for="field14">Comments:</label>
+          <textarea name="Comments" id="field14"></textarea>
```

Comment:
name should not start from capital letter
```suggestion
          <textarea name="comments" id="field14"></textarea>
```